packages/shared/.env.example

```
@@ -0,0 +1,5 @@
+# Database (Turso/LibSQL)
+DATABASE_URL="http://localhost:8080"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)